src/selfie_avoiding_obstacles/include/selfie_avoiding_obstacles/road_obstacle_detector.h

```diff
@@ -71,24 +73,25 @@ class Road_obstacle_detector
   float left_lane_;
 
   float max_speed_;
-  float safe_speed_;
+  float slowdown_speed_;
+  float lane_change_speed_;
   std_msgs::Float64 speed_message_;
 
-  float maximum_distance_to_obstacle_; // to avoid changing lane too early
-  float maximum_length_of_obstacle_;
+  float max_distance_to_obstacle_; // to avoid changing lane too early
```

Comment:
Nie mamy max_distance_to_slowdown. To on jest sprawdzany do proof_slowdown. Gdy proof_slowdown == num_proof_to_slowdown i przeszkoda jest w max_distance_to_obstacle (max_distance_to_overtake) to zmieniamy pas